nixos/modules/system/boot/clevis.md

```diff
@@ -0,0 +1,49 @@
+# Clevis {#module-boot-clevis}
+
+[Clevis](https://github.com/latchset/clevis)
+is a framework for automated decryption policy of ressources.
+Clevis allows for secure unattended disk decryption during boot, using decryption policies that must be satisfied for the data to decrypt.
+
+
+## Create a secret {#module-boot-clevis-create-secret}
+
+Secrets have to be created through the clevis command line. 3 types of policies are supported:
+
+1) TPM policies
+
+Secrets are binded against the presence of a TPM2 device, for example:
```

Comment:
Typo/improper conjugation: binded => bound. 

And is this the correct word? The `clevis` verb is `encrypt` and it talks about a "pin". 

> First, is the pin. In clevis terminology, a pin is a plugin which implements automated decryption. We simply pass the name of a pin here.

I note that this terminology doesn't use that word. Should it? The word "pinned" makes quite a bit of sense to me, and likely should replace all uses of "bound" / "binded" throughout.